src/sql/magic.py

```diff
@@ -72,6 +72,13 @@ class RenderMagic(Magics):
     @telemetry.log_call("sqlrender")
     def sqlrender(self, line):
         args = parse_argstring(self.sqlrender, line)
+        warnings.warn(
+            "'%sqlrender' will be deprecated soon, "
+            f"please use '%sqlcmd snippets {args.line[0]}' instead. "
+            "For documentation, follow this link : "
+            "https://jupysql.ploomber.io/en/latest/api/magic-snippets.html#id1",
```

Comment:
<img width="1085" alt="image" src="https://github.com/ploomber/jupysql/assets/103026689/31cd3aff-125c-47d8-97ae-c7fd6eb78bd1">

I think we can format it in a way it will be easier to read. maybe something like this
```
        warnings.warn(
            "\n'%sqlrender' will be deprecated soon, "
            f"please use '%sqlcmd snippets {args.line[0]}' instead. "
            "\n\nFor documentation, follow this link : "
            "https://jupysql.ploomber.io/en/latest/api/magic-snippets.html#id1",
            FutureWarning,
        )
```
I broke this entire text block into rows.